### DISCUSSION TITLE
mbed-mqtt

### DISCUSSION THREAD

**Discussion by chrissnow** [mbed-ce/mbed-os#375] [General]
I have been working on tidying up the MQTT library below inc the PR's that were suggested.
https://github.com/ARMmbed/mbed-mqtt
If you can fork it to CE I will submit a PR.

**Comment by JohnK1987** [comment]
Hello Chris, for libraries we have separeted organization https://github.com/mbed-ce-libraries-examples

**Reply by chrissnow** [reply]
Happy to push to there but I think you need to fork first?

**Reply by JohnK1987** [reply]
I already send you the invite to the organization. Please read https://github.com/mbed-ce-libraries-examples/.github/blob/main/profile/README.md and then you can do it by your self.
